src/openrct2-ui/windows/Ride.cpp

```diff
@@ -2658,13 +2658,14 @@ static void window_ride_main_invalidate(rct_window* w)
  *
  *  rct2: 0x006AF10A
  */
-static rct_string_id window_ride_get_status_overall_view(rct_window* w, void* arguments)
+static rct_string_id window_ride_get_status_overall_view(rct_window* w)
```

Comment:
This is going backwards. We want to eventually stop using the global variables completely this is readding the global

Reply:
Fixed, do squash on merge